lib/generate.js

```diff
@@ -138,22 +127,26 @@ function generateLayoutInternal(options, callback) {
             if (err) return callback(err);
             if (!image.width() || !image.height()) return callback(null, null);
 
-            let q = new queue();
-            q.defer(encodeImage, image);
-
-            // Stretch metadata only needs to be extracted for the data layout JSON (when options.format is true)
+            // For the data layout JSON (when options.format is true)
+            // - We need to extract stretch metadata
```

Comment:
This comment looks like it is dangling from a bulleted list that perhaps no longer exists

Reply:
Correct, fixed in ecbb720